File: MyCastleSpa/src/app/home/models/board.ts
export class Board {

    public id: number;

    public name: string;

    public imagePath: string;

    public isFavorite: boolean;

    public color: string;

    constructor(){
        
        this.id = 0;

        this.name = '';

        this.imagePath = '';

        this.color = '#3d18e7';

        this.isFavorite = false;
    }
}

File: Common/Entity/ProfileService/Board.cs
using System.ComponentModel.DataAnnotations;

namespace Common.Entity.ProfileService
{
    public class Board
    {
        [Required]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Color { get; set; }

        [Required]
        public string ImagePath { get; set; }

        public bool IsFavorite { get; set; }
    }
}
